src/mercury/components/terminal-link.jsx

```diff
@@ -78,7 +75,7 @@ class TerminalLink extends React.Component<Props, State> {
   getCurrentInputCommand() {
     const history = this.props.terminal.history;
     return history[this.state.historyIndex];
-  };
```

Comment:
Why are some of these functions standard functions without semi colons and some are variables assigned to a function lambda (like onPaste) above? We should standardize that at some point.

Reply:
It's because class methods aren't bound normally, so the value of `this` can be undefined in callback methods. The assigned lambda is experimental syntax that automatically binds properly. More about that here: https://reactjs.org/docs/handling-events.html (about halfway down the page).